gratipay/models/exchange_route.py

```diff
@@ -53,12 +49,14 @@ def from_address(cls, participant, network, address):
         return r
 
     @classmethod
-    def insert(cls, participant, network, address, error='', fee_cap=None, cursor=None):
+    def insert(cls, participant, network, address, error='', fee_cap=None, cursor=None,
+                                                                                 is_deleted=False):
```

Comment:
as we don't really "delete' the route, shall we name it (and in the database) something like 'active'?